scripts/ISM/ISM.py

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python3
+# # -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# limitations under the License.
+"""Create ISM catalogs.
+
+This script is used to convert Australian government ISM into OSCAL formats.
+It does this via pulling the ISM xml doc and creating:
+
+1 Catalog for all the controls
+4 profiles (Official, protected, secret, TS)
+
+Ideally this would be a cron job based script, however, as ACSC publish revisions
+with specific names this would need to be discovered by crawling. This will be a potential future enhancement.
+
+This script pulls down teh controls in a 'dumb' way from the xml to get the actual controls. A full featured catalog
```

Comment:
typo with teh.
Anybody may land on this demo so it would be good to explain "ISM" and give a bit of context.

Reply:
Done.